api/app/utils/generate_fuel_cog.py

```diff
@@ -0,0 +1,118 @@
+import os
+from tempfile import TemporaryDirectory
+
+import numpy as np
+from osgeo import gdal
+
+from wps_shared import config
+from wps_shared.geospatial.wps_dataset import WPSDataset
+from wps_shared.utils.s3 import set_s3_gdal_config
+
+bucket = config.get("OBJECT_STORE_BUCKET")
+
+# Input
+INPUT_GEOTIFF = f"/vsis3/{bucket}/psu/rasters/fuel/FM_FUEL_TYPE_GRID_BC_2025_500m_BC_ONLY.tif"
+# Output
+OUTPUT_PATH = "/Users/breedwar/Downloads/fbp2025_500m_cog.tif"
```

Comment:
We may want to make the output path relative to the script path, so we don't hardcode machine specifics.